dtos/http_response.go

```diff
@@ -0,0 +1,31 @@
+package dtos
+
+type FormCreationResponse struct {
+	Mesaage string `json:"message"`
```

Comment:
- what is the required for this? if we're having it in every response should we have a structured response for every response? 
```js
{
   message: <string>
   data: {
       ...api response
   }
}
```